### PR TITLE
Marks Mac dart_plugin_registry_test to be flaky

### PR DESCRIPTION
**Pull request by keyonghan**
Issue link: https://github.com/flutter/flutter/issues/99940

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -2594,6 +2594,7 @@ targets:
 
   - name: Mac dart_plugin_registry_test
     recipe: devicelab/devicelab_drone
+    bringup: # Flaky https://github.com/flutter/flutter/issues/99940
```
Comment (godofredoc) [inline .ci.yaml:2597]: are we missing `True` before the comment?

Comment (keyonghan) [reply .ci.yaml]: Yes! Added.

**Review by godofredoc — APPROVED** [review]

**Comment by skia-gold** [comment]
Gold has detected about 7 new digest(s) on patchset 1.
View them at https://flutter-gold.skia.org/cl/github/99941

**Comment by blasten** [comment]
thanks! I was in process of sending this PR. 

https://github.com/flutter/flutter/pull/99936 is adding more logging

**Review by blasten — APPROVED** [review]

**Comment by skia-gold** [comment]
Gold has detected about 9 new digest(s) on patchset 2.
View them at https://flutter-gold.skia.org/cl/github/99941

**Comment by keyonghan** [comment]
Manually submit to unblock the tree.

**Pull request merged**